chef/cookbooks/nova/recipes/compute.rb

```diff
@@ -423,3 +429,36 @@ def set_boot_kernel_and_trigger_reboot(flavor='default')
 execute "trigger-nova-own-az-config" do
   command "true"
 end
+
+# Set iptables rules for blocking VNC Access for all but the nova-controller node.
+bash "nova_compute_vncblock_reject_all" do
+  code <<-EOH
+    iptables -N VNCBLOCK
+    iptables -I INPUT -j VNCBLOCK -p tcp --match multiport --dports 5900:15900
+    iptables -F VNCBLOCK
+    iptables -I VNCBLOCK -p tcp \
+        -m connbytes --connbytes 0:1024 \
+        --connbytes-dir both --connbytes-mode bytes \
```

Comment:
man iptables-extensions says on this topic
```
NOTE that for connections which have no accounting information, the match will always return false.
```
and cat /proc/sys/net/netfilter/nf_conntrack_acct only shows a 1 for SLE11, but not SLE12 by default, so I guess if we wanted to use this, we would need to enable accounting.